modin/pandas/test/utils.py

```diff
@@ -454,6 +454,8 @@
 # the type of this exceptions are the same
 io_ops_bad_exc = [TypeError, FileNotFoundError]
 
+default_to_pandas_ignore_string = "default:.*defaulting to pandas.*:UserWarning"
```

Comment:
Why is `default` prefix here, but not `error` as in setup.cfg?

Reply:
We use the line in `setup.cfg` to treat all user warnings as errors in tests. This line is for doing the opposite, ignoring the error raised by defaulting to pandas. We use it as a temporary fix in some tests where there are too many errors to catch until we fix issues like #3955

Reply:
Ok, I got it, thanks! Two other questions from me:

1. Why do we need to set `filterwarnings` twice - in `modin/conftest.py` and in several test files? Isn't setting `filterwarnings` in `modin/conftest.py` enough?
2. Is the prefix `default` related to pytest?

Reply:
@mvashishtha, ping

Reply:
> Why do we need to set `filterwarnings` twice

 `conftest.py` is using `filterwarnings` to ignore the default to pandas under BaseOnPython execution for all tests. Several tests also have many uncaught warnings of defaulting to pandas, and we will address those in issues like #3955. For now, we use `filterwarnings`to ignore all such warnings in those tests.

> The prefix `default` is unrelated to pytest; it just [prints the warnings](https://docs.python.org/3/library/warnings.html#the-warnings-filter).

Reply:
@mvashishtha, thank you!